ros2topic/ros2topic/verb/list.py

```diff
@@ -34,19 +46,31 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--include-hidden-topics', action='store_true',
             help='Consider hidden topics as well')
+        parser.add_argument(
+            '-v', '--verbose', action='store_true',
+            help='list full details about each topic')
```

Comment:
Perhaps the command should change to something like `--show-connections` (maybe there's a better name) or be broken into `--show-subscriptions` and `--show-publishers`. You could also have a `--verbose` command that enables everything, but it would be nice to have the ability to enable things individually.

Reply:
I prefer to add this feature in another PR.